common/src/main/java/com/microsoft/identity/common/internal/cache/CacheKeyValueDelegate.java

```diff
@@ -208,6 +209,12 @@ public String generateCacheValue(Credential credential) {
         return result;
     }
 
+    @Override
+    public String generateAuthorityValidationMetadataKey(final String environment)
```

Comment:
My preference is to move this to another class... as this is used by the token cache